sql/moz-fx-data-shared-prod/internet_outages/global_outages_v1/query.sql

```diff
@@ -162,18 +176,32 @@ health_data_sample AS (
   WHERE
     DATE(submission_timestamp) = @submission_date
   GROUP BY
-    1,
-    2,
-    3,
-    4
+    country,
+    city,
+    geo_subdivision1,
+    geo_subdivision2,
+    `datetime`,
+    client_id
 ),
 health_data_aggregates AS (
   SELECT
     country,
     -- If cities are either '??' or NULL then it's from cities we either don't
     -- know about or have a population less than 15k. Just rename to 'unknown'.
     IF(city = '??' OR city IS NULL, 'unknown', city) AS city,
-    datetime,
+    IF(
+      geo_subdivision1 = '??'
+      OR geo_subdivision1 IS NULL,
+      'missing',
```

Comment:
Any specific reason for this to be `missing` instead of `unknown` as well? (we should document it in the above commend and in the query docs regardless though :-D )

Reply:
this is more a placeholder to make sure the join condition in the final statement works as intended.